Modules/CalcDefence-3_0.lua

```diff
@@ -29,9 +29,14 @@ function calcs.hitChance(evasion, accuracy)
 	return m_max(m_min(round(rawChance), 100), 5)	
 end
 
--- Calculate physical damage reduction from armour
+-- Calculate physical damage reduction from armour, float
+function calcs.armourReductionF(armour, raw)
+	return (armour / (armour + raw * 10) * 100)
+end
+
+-- Calculate physical damage reduction from armour, int
 function calcs.armourReduction(armour, raw)
-	return round(armour / (armour + raw * 10) * 100)
+	return round(calcs.armourReductionF(armour, raw))
```

Comment:
Is there a reason this round function is only used for player damage reduction but not enemy damage reduction?

Reply:
The original damage reduction was rounded, and really only used for display purposes since no further calculations are being done with player armour.
I kept it there to avoid breaking any existing functionality.